manifests/profile/www_lib/vhosts/apps_lib.pp

```diff
@@ -190,10 +169,8 @@
         # 2016-08-29 skorner per nancymou
         rewrite_rule => '^/islamic	http://guides.lib.umich.edu/islamicmss/find 	[redirect=permanent,last]'
       },
-      {
-        rewrite_cond => '%{REQUEST_URI} !^/openid-connect',
-        rewrite_rule => '^(/instruction/request.*)$ https://sali1.lib.umich.edu:8443$1 [P]',
-      },
+
+        # No More SALI redirect in favor of stand-alone drupal app
```

Comment:
We still need to include a basic redirect to the new site at the former URL root:
`rewrite_rule => '^(/instruction/request.*)$ https://sali.lib.umich.edu/ [redirect=permanent,last]'`